FOSOAuthServerBundle.php

```diff
@@ -16,22 +16,31 @@
 use FOS\OAuthServerBundle\DependencyInjection\FOSOAuthServerExtension;
 use FOS\OAuthServerBundle\DependencyInjection\Security\Factory\OAuthFactory;
 use FOS\OAuthServerBundle\DependencyInjection\Compiler\GrantExtensionsCompilerPass;
+use Symfony\Bundle\SecurityBundle\DependencyInjection\SecurityExtension;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\HttpKernel\Bundle\Bundle;
 use Symfony\Component\HttpKernel\Kernel;
 
 class FOSOAuthServerBundle extends Bundle
 {
+    /**
+     * @example '2.1.0'
+     * @var string
+     */
+    private $kernelVersion;
+
     public function __construct()
     {
+        $this->kernelVersion = Kernel::VERSION;
         $this->extension = new FOSOAuthServerExtension();
     }
 
     public function build(ContainerBuilder $container)
     {
         parent::build($container);
 
-        if (version_compare(Kernel::VERSION, '2.1', '>=')) {
+        if (version_compare($this->kernelVersion, '2.1', '>=')) {
```

Comment:
Why this indirection?

Reply:
I needed a way to be able to test behaviour with different versions. Here's the test:

https://github.com/dinamic/FOSOAuthServerBundle/blob/feature/unit-tests-1.6/Tests/FOSOAuthServerBundleTest.php#L37

I may have named the test the wrong way, but Kernel version is indication of the framework version being used, right?

I had that presumption when naming the tests and was thinking that I can even delete this if statement, but wasn't sure whether it won't break functionality.